src/version.h

```diff
@@ -19,22 +19,22 @@ static const int INIT_PROTO_VERSION = 209;
 static const int GETHEADERS_VERSION = 70077;
 
 //! disconnect from peers older than this proto version
-static const int MIN_PEER_PROTO_VERSION = 70066;
+static const int MIN_PEER_PROTO_VERSION = 70103;
 
 //! minimum peer version accepted by DarksendPool
-static const int MIN_POOL_PEER_PROTO_VERSION = 70103;
+static const int MIN_POOL_PEER_PROTO_VERSION = 70200;
 
 //! minimum peer version for masternode budgets
-static const int MIN_BUDGET_PEER_PROTO_VERSION = 70103;
+static const int MSG_GOVERNANCE_PEER_PROTO_VERSION = 70200;
 
 //! minimum peer version for masternode winner broadcasts
 static const int MIN_MNW_PEER_PROTO_VERSION = 70103;
 
 //! minimum peer version that can receive masternode payments
 // V1 - Last protocol version before update
 // V2 - Newest protocol version
-static const int MIN_MASTERNODE_PAYMENT_PROTO_VERSION_1 = 70066;
-static const int MIN_MASTERNODE_PAYMENT_PROTO_VERSION_2 = 70103;
+static const int MIN_MASTERNODE_PAYMENT_PROTO_VERSION_1 = 70103;
+static const int MIN_MASTERNODE_PAYMENT_PROTO_VERSION_2 = 70200;
```

Comment:
Shouldnt it be '70201'? '70200' was a internal testnet version that will never make it to mainnet.